### DISCUSSION TITLE
"npx create-remix --template" does not work

### DISCUSSION THREAD

**Discussion by bholguin** [remix-run/remix#3029] [Q&A]
I created this project two week ago, and template was working, but this week i tried generate my custom template and i have this error.

`Creating your app…🚨 There was a problem fetching the file from GitHub. The request responded with a 404 status. Please try again later.`

i am using this command

`npx create-remix@latest --template https://github.com/bholguin/remix-stack-custom.git`

i can see that remix updated their version one week ago but it does not work in none version

i tried too 

`npx create-remix@1.3.3 --template https://github.com/bholguin/remix-stack-custom.git`

**Comment by MichaelDeBoey** [comment]
Removing the `.git` from the end of the URL makes it work

```sh
npx create-remix@latest --template https://github.com/bholguin/remix-stack-custom
```
